tdrs-backend/docs/security-tracking/sprint4_frontend_09292020.md

```diff
@@ -0,0 +1,39 @@
+# Sprint 4 Frontend Vulnerability Report (09/29/2020) 
```

Comment:
While I don't have any objections to merging this in, just noting that Sprint 4 was a while back, and I don't believe we're writing up reports for every Synk update anymore (see #113). So these update docs could be safely removed from my point of view.

Reply:
I can remove it - this was something created much earlier

Reply:
@alexsoble This has been removed from the PR

Reply:
Thanks! I'm also seeing a `sprint5_backend_10092020.md` file in the same folder that can be removed as well.

Reply:
Ok, that's been removed also

Reply:
👍  Thank you Carl!